### PR TITLE
add revertrisk multilingual

### PR DESCRIPTION
**Pull request by mercelisvaughan**
Add ability to make request to revert risk multilingual with validation with updated README

# get prediction

```
model = RevertRiskMultilingualModel()
payload = {
    "lang": "en",
    "rev_id": 123456
}
result = model.request(payload=payload)
print(result)

```

<details>

<summary>Response</summary>

```
{'model_name': 'revertrisk-language-agnostic', 'model_version': '3', 'wiki_db': 'enwiki', 'revision_id': 123456, 'output': {'prediction': False, 'probabilities': {'true': 0.25512129068374634, 'false': 0.7448787093162537}}}
{'model_name': 'revertrisk-multilingual', 'model_version': '4', 'wiki_db': 'enwiki', 'revision_id': 123456, 'output': {'prediction': False, 'probabilities': {'true': 0.3156708782344262, 'false': 0.6843291217655738}}}
```

</details>

### CODE REVIEW

File: liftwing/models/revertrisk_multilingual.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        """
+        if self.payload_model:
+            _ = self.payload_model(**payload)
+
+        if headers is None:
+            headers = {}
+
+        response = requests.post(self.b, json=payload, headers=headers)
+
+        return response.json()
```
Comment (isaranto) [inline liftwing/models/revertrisk_multilingual.py]: This is not needed as the request function of the parent class `LiftwingModel` would do the job.
```suggestion
```

**Review by isaranto — CHANGES_REQUESTED** [review]

**Comment by isaranto** [comment]
Nice work! 
We just need to remove the request function and fix the CI checks.
For more information on what to fix check out the [output of the GH action](https://github.com/wikimedia/liftwing-python/actions/runs/9506801086/job/26204843986?pr=16).
To run pre-commit locally you can run:

```
pip install pre-commit
pre-commit run --all-files
```

**Comment by isaranto** [comment]
@mercelisvaughan in order to be able to import the model with `from liftwing import RevertRiskMultilingualModel` we'll need to add it to the __ini__.py file like we did for the other models.

File: liftwing/models/readability.py
Comment (kevinbazira) [inline liftwing/models/readability.py:1]: This file is adding the readability model without ModelMetadata. The readability model has already been added in #11 with ModelMetadata.

**Review by kevinbazira — COMMENTED** [review]
Most of it looks good, besides `liftwing/models/readability.py` which seems redundant.

File: liftwing/models/readability.py
Comment (isaranto) [reply liftwing/models/readability.py:1]: You're right I missed this one!

**Review by kevinbazira — APPROVED** [review]
LGTM!

**Review by isaranto — APPROVED** [review]

**Pull request merged**